internal/pkg/plugin/plugin_installer.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 const (
-	programNotFoundErrorMsg      = "unable to find %s; check that it is installed in a directory in your $PATH"
+	programNotFoundErrorMsg      = "unable to find %s"
+	programNotFoundSuggestions   = "check that it is installed in a directory in your $PATH"
```

Comment:
Suggestions should be full sentences
```suggestion
	programNotFoundSuggestions   = "Check that it is installed in a directory in your $PATH."
```

Reply:
> Suggestions should be full sentences

I know, this was just a copy paste error. Thanks for catching it!